static/js/panels/ProfileForm.tsx

```diff
@@ -75,6 +132,64 @@ const ProfileForm: FC = () => {
                 }
                 stacked
               />
+              <Accordion
+                titleElement="h3"
+                sections={[
+                  {
+                    content: (
+                      <>
+                        <NicDeviceSelector
+                          notify={notify}
+                          setNicDevice={setNicDevice}
+                        />
+                        <hr />
+                        <DiskDeviceSelector
+                          notify={notify}
+                          setDiskDevice={setDiskDevice}
+                        />
+                      </>
+                    ),
+                    title: "Devices",
+                  },
+                  {
+                    content: (
+                      <>
+                        <LimitSelector
+                          cpuLimit={cpuLimit}
+                          setCpuLimit={setCpuLimit}
+                          memoryLimit={memoryLimit}
+                          setMemoryLimit={setMemoryLimit}
+                          measure={measure}
+                          setMeasure={setMeasure}
+                        />
+                      </>
+                    ),
+                    title: "Limits",
+                  },
+                  {
+                    content: (
+                      <>
+                        <CloudInitConfig
```

Comment:
Couldn't we have just one input for the cloud init yaml with the three blocks prefilled but empty?

Reply:
We could, but I don't think we should. They have separate config keys, so grouping them in one single `YamlEditor` means that we should then build a parser to split them when we build the `config` object. Not optimal and very error prone. And user-wise it's also not desirable, because it confuses the user by making them think that they are a single block of config, while they aren't. They are separate configs that can be set or not independently, and I think this should be apparent to the user.